gopkg/translate.py

```diff
@@ -0,0 +1,8 @@
+def translate(pkg: str) -> str:
```

Comment:
are we planning to use types? @msrb? please make sure the CI will use the right Python version then

Reply:
I decided to leave them out.